packages/psss/src/api/mutations.js

```diff
@@ -3,7 +3,7 @@
  * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
  */
 
-import { queryCache, useMutation } from 'react-query';
+import { useQueryClient, useMutation } from 'react-query';
```

Comment:
`queryCache` has been replaced by `useQueryClient` hook, see: https://tanstack.com/query/v4/docs/react/guides/migrating-to-react-query-3#the-usequerycache-hook-has-been-replaced-by-the-usequeryclient-hook